packages/web-components/fast-components/src/slider/slider.styles.ts

```diff
@@ -29,135 +38,174 @@ export const sliderStyles: (
-    }
-    :host([orientation="horizontal"]) .positioning-region {
-        position: relative;
-        margin: 0 8px;
-        display: grid;
-        grid-template-rows: calc(var(--thumb-size) * 1px) 1fr;
-    }
-    :host([orientation="vertical"]) .positioning-region {
-        position: relative;
-        margin: 0 8px;
-        display: grid;
-        height: 100%;
-        grid-template-columns: calc(var(--thumb-size) * 1px) 1fr;
-    }
-
-    :host(:${focusVisible}) .thumb-cursor {
-        box-shadow: 0 0 0 2px ${fillColor}, 0 0 0 4px ${focusStrokeOuter};
-    }
-
-    .thumb-container {
-        position: absolute;
-        height: calc(var(--thumb-size) * 1px);
-        width: calc(var(--thumb-size) * 1px);
-        transition: all 0.2s ease;
-        color: ${neutralForegroundRest};
-        fill: currentcolor;
-    }
-    .thumb-cursor {
-        border: none;
-        width: calc(var(--thumb-size) * 1px);
-        height: calc(var(--thumb-size) * 1px);
-        background: ${neutralForegroundRest};
-        border-radius: calc(${controlCornerRadius} * 1px);
-    }
-    .thumb-cursor:hover {
-        background: ${neutralForegroundRest};
-        border-color: ${neutralStrokeHover};
-    }
-    .thumb-cursor:active {
-        background: ${neutralForegroundRest};
-    }
-    :host([orientation="horizontal"]) .thumb-container {
-        transform: translateX(calc(var(--thumb-size) * 0.5px)) translateY(calc(var(--thumb-translate) * 1px));
-    }
-    :host([orientation="vertical"]) .thumb-container {
-        transform: translateX(calc(var(--thumb-translate) * 1px)) translateY(calc(var(--thumb-size) * 0.5px));
-    }
-    :host([orientation="horizontal"]) {
-        min-width: calc(var(--thumb-size) * 1px);
-    }
-    :host([orientation="horizontal"]) .track {
-        right: calc(var(--track-overhang) * 1px);
-        left: calc(var(--track-overhang) * 1px);
-        align-self: start;
-        height: calc(var(--track-width) * 1px);
-    }
-    :host([orientation="vertical"]) .track {
-        top: calc(var(--track-overhang) * 1px);
-        bottom: calc(var(--track-overhang) * 1px);
-        width: calc(var(--track-width) * 1px);
-        height: 100%;
-    }
-    .track {
-        background: ${neutralStrokeRest};
-        position: absolute;
-        border-radius: calc(${controlCornerRadius} * 1px);
-    }
-    :host([orientation="vertical"]) {
-        height: calc(var(--fast-slider-height) * 1px);
-        min-height: calc(var(--thumb-size) * 1px);
-        min-width: calc(${designUnit} * 20px);
-    }
-    :host([disabled]), :host([readonly]) {
-        cursor: ${disabledCursor};
-    }
-    :host([disabled]) {
-        opacity: ${disabledOpacity};
-    }
-`.withBehaviors(
+        ${display("inline-grid")} :host {
+            --thumb-size: calc((${heightNumber} / 2) + ${designUnit});
+            --thumb-translate: calc(var(--thumb-size) * 0.5);
+            --track-overhang: calc((${designUnit} / 2) * -1);
```

Comment:
These only seem to be used below directly and not in other calculations. You could turn them into px values here and save some bytes below.

```suggestion
            --thumb-translate: calc(var(--thumb-size) * 0.5px);
            --track-overhang: calc((${designUnit} / 2) * -1px);
```

Reply:
This change seems to collapse the track. I will leave it as is, at least for now.